src/views/Phrases.vue

```diff
@@ -84,68 +94,69 @@ export default {
     <h1 class="page-title">Phrases</h1>
 
     <div class="phrase-input-category-box is-centered">
-      <br>
+      <br />
       <div class="phrase-input-box">
-        <br>
+        <br />
         <strong>Input Example:</strong>
-        <br><br>
+        <br />
+        <br />
         <div>
-          <img class="start-quote-img" align="left" src="../assets/start-quote.svg" alt="start-quote.svg">
+          <img
+            class="start-quote-img"
+            align="left"
+            src="../assets/start-quote.svg"
+            alt="start-quote.svg"
+          />
         </div>
-        <input
-          type="text"
-          placeholder="Phrase Example"
-          class="phrase-input"
-          v-model="phrase"
-        >
-        <br>
+        <input type="text" placeholder="Phrase Example" class="phrase-input" v-model="phrase" />
+        <br />
         <div>
-          <img class="end-quote-img" align="right" src="../assets/end-quote.svg" alt="end-quote.svg">
+          <img
+            class="end-quote-img"
+            align="right"
+            src="../assets/end-quote.svg"
+            alt="end-quote.svg"
+          />
         </div>
-        <br>
-        <br>
-        <br>
+        <br />
+        <br />
+        <br />
       </div>
 
       <div v-if="this.phrase != ''" class="phrase-input-box">
-        <br>
-        <div><strong>Query:</strong></div>
+        <br />
         <div>
-          <button v-for="(tokenObj, index) in this.buttonData"
-                  v-bind:key="index"
-                  v-bind:class="{ 'data-button':true,
-                                  'round-outlined-button':true,
-                                  'selected': tokenObj.isSelected
-                  }"
-                  v-on:click="toggleIsSelected(tokenObj)">
-            {{ buttonData[index].token }}
-          </button>
+          <strong>Query:</strong>
+        </div>
+        <div :key="tokenRerenderKey">
+          <button
+            class="data-button round-outlined-button"
+            v-for="(tokenObj, index) in this.buttonData"
```

Comment:
BTW in the HTML Vue statements, ``this`` is not required:
```
v-for="(tokenObj, index) in buttonData"
```

Reply:
Got it!